models/xero__general_ledger.sql

```diff
@@ -1,3 +1,41 @@
+/*
+in order to generate trancking_category_name as a column we set a dictionary
+dictionary has the following appareance{'tracking_category_id' : [tuple of available id], 'tracking_category_name': [tuple of available names]}
+position match for id and name since the dictionary was generated with a sql statement 
+*/
+
+{% set tracking_category_sql %}
+    select 
+        distinct tracking_category_id,
+        regexp_replace(lower(name), r'\W', '_')   as tracking_category_name
```

Comment:
Same comment as in the dbt-runner https://github.com/GetAleph/dbt-runner/pull/317/files#r924014264. Also, do we need this duplicated code?

Reply:
I am not sure how to avoid the duplicate code since it is happening in 2 different packages.